src/components/Input/Input.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from "react";
+import { InputType, InputElementProps } from "../Input/type";
+import classes from './Input.module.scss';
+
+const Input: FC<InputElementProps> = ({type, placeHolder, name, required, id, options=[], value}: InputElementProps) => {
+  const [enteredValue, setValue] = useState<string>('');
+  const [isValid, setIsValid] = useState<boolean | null>(true);
+
+  const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = e.target.value;
+    setValue(newValue);
+    setIsValid(validation(type, newValue));
+  }
+
+  return (
+    <div className={classes.inputContainer}>
+      {!isValid? errorElement(type, isValid) : ``}
+      {type === InputType.SELECT &&
```

Comment:
make this two conditions in one ternary statement

Reply:
Ternary statement emplemented